variables.tf

```
@@ -0,0 +1,38 @@
+variable "subscriptions" {
+  type        = map(string)
+  description = <<-DOC
+  ```
+  [
+    "<subscription_name>" = "<subscription_id>"    (list of subscriptions to recieve default group assignments) 
+  ]
+  ```
+  DOC
```

```suggestion
  description = "Subscriptions to create default (Owner, Contributor and Reader) Groups and assignments. Key of the map should be the name of the subscription. This is used to generate the name of the group by prefixing with `SUB_` and add the role as suffix. The value of the map should be the subscription id to identify the subscription."
```

isn't it more reasonable to just get the subscription ids and get the name ourself?

---

We have the subscription map available in the governance module anyways and are saving datasources.